addons/maintenance/models/maintenance.py

```diff
@@ -190,7 +190,9 @@ def _read_group_category_ids(self, categories, domain):
         """ Read group customization in order to display all the categories in
             the kanban view, even if they are empty.
         """
-        category_ids = categories._search([], order=categories._order, access_rights_uid=SUPERUSER_ID)
+        # search with global ir.rules only
+        search_domain = self.env['ir.rule']._compute_domain(categories._name) or []
+        category_ids = categories.sudo()._search(search_domain, order=categories._order)
```

Comment:
IMO, should it be the default behavior of `_read_group_expand_full`? Because, IMO, we never want to filter out inaccessible (from no-global ir.rules) records when we group by a `Many2one` ?

Reply:
I see what you mean.  That kind of expansion is really bad practice IMO.  The ones I consider good practice do search with a domain like "`id` in found records OR some condition including extra records".  That's why we are not even sure that using (global) access rules is a good idea.